.github/workflows/release.yaml

```diff
@@ -30,3 +31,18 @@ jobs:
 
       - name: Push Image
         run: OSS_REGISTRY="ghcr.io/oracle" VERSION="${{ github.ref_name }}" make docker-push-all
+
+      - name: Make manifests
+        run: RELEASE=${{ github.ref_name }} make manifests
+
+      - name: Get current release
+        id: get_release
+        uses: bruceadams/get-release@v1.3.2
```

Comment:
I am not sure if this is allowed gitactions by OGHO. please get it reviewed by @Djelibeybi

Reply:
@Djelibeybi, 
Could you pls review this git action and confirm whether we can use this or not?

Reply:
It's not in the list of allowed actions. You'd either have to replace it, fork it or request approval for it.